### PR TITLE
Add use-package usage to documentation

### PR DESCRIPTION
**Pull request by alissonperez**
Add instructions to README to setup use-package with editorconfig.

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,14 @@ Alternatively, you can find the package available on
 [MELPA](http://melpa.org/#/editorconfig) and [MELPA Stable](http://stable.melpa.org/#/editorconfig)
 ([The Marmalade package](http://marmalade-repo.org/packages/editorconfig) is deprecated).
 
+Or if you use [**use-package**](https://www.emacswiki.org/emacs/UsePackage):
+
+```emacs-list
+(use-package editorconfig
+  :ensure t
+  :init
+  (add-hook 'prog-mode-hook 'editorconfig-mode))
```
Comment (xuhdev) [inline README.md]: Maybe other text modes should also be included?

Comment (10sr) [inline README.md]: `editorconfig-mode` is a global minor mode, so I think this should be just `(editorconfig-mode 1)`.

Comment (xuhdev) [inline README.md]: The reason I pointed out it is because normally people don't want editorconfig when they open up PNG files, for example.

Comment (10sr) [inline README.md]: I see.
To enable editorconfig only for text modes, we can use expressions like `(add-hook 'mode-hook 'editorconfig-apply)` 🐵

Comment (alissonperez) [inline README.md]: @xuhdev and @10sr , I've made this commit https://github.com/editorconfig/editorconfig-emacs/pull/103/commits/6f55f00fc0c8920c5c7b443b09f74e83b7d6ba8c to make some changes.

Please, tell me if you have any other suggestion.

Thanks!

**Comment by 10sr** [comment]
Thanks:exclamation: 

However, I think with this configuration editorconfig properties will not be applied to the first file you opened after starting Emacs... (editorconfig will be enabled *after* opening the first file, so not work for that file)
Additionally, I think it would be better to add a code similar to the one used above (manual MELPA install).
So instead something like this might be better, IMO.

```
:config
(editorconfig-mode 1)
```

What do you think?

**Comment by 10sr** [comment]
Also, I'll add a example to enable editorconfig for some specific modes later :cake:

**Comment by 10sr** [comment]
Marging anyway.
 Thanks! 👍

**Pull request merged**